consensus/scp/core/src/msg.rs

```diff
@@ -629,9 +633,10 @@ impl<V: Value, ID: GenericNodeId> fmt::Display for Msg<V, ID> {
         };
 
         // Returns "<set.len, hash(set)>".
+        let hasher_builder = HasherBuilder::default();
         let format_b_tree_set = |b_tree_set: &BTreeSet<V>| {
             let hash = {
-                let mut hasher = DefaultHasher::new();
+                let mut hasher = hasher_builder.build_hasher();
```

Comment:
This needs a sanity check - there are an exceptional volume of msgs that get created - if we're changing this, there should be mindfulness towards the impact.

Reply:
Fair point. This is in `Display`, so it's fairly safe to change; if we're logging all messages we're in trouble anyway.